app/views/content/non-uk-teachers/_why-teach.html.erb

```diff
@@ -13,7 +13,6 @@
       </p>
 
       <ul>
-        <li><%= link_to("Read Brooke's story about coming to England to teach", "/blog/my-move-from-canada-to-teach-in-england") %></li>
```

Comment:
Should we keep this for now (and the blog itself and the other international blog), given international content is not moving yet @philspray ?